x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/edit_field/advanced_settings_wrapper.tsx

```diff
@@ -20,13 +21,20 @@ export const AdvancedSettingsWrapper = ({ children }: Props) => {
 
   return (
     <div className="mappingsEditor__editField__advancedSettings">
-      <div>
-        <EuiButtonEmpty onClick={toggleIsVisilbe}>
-          {isVisible ? 'Hide' : 'Show'} advanced settings
-        </EuiButtonEmpty>
-      </div>
+      <EuiSpacer size="m" />
+
+      <EuiButtonEmpty onClick={toggleIsVisilbe} flush="left">
```

Comment:
looks like this was pre-existing but you might change `toggleIsVisilbe` to `toggleIsVisible`